### PR TITLE
Switched to a /:entity/:format/:version/:draft/ URL scheme

### PR DESCRIPTION
**Pull request by kunagpal**
- [x] Remove the `DO NOT MERGE` prefix when all dependencies have been assessed.

### CODE REVIEW

File: dist/index.mustache [hunk 1]
```diff
@@ -5,8 +5,8 @@
       <meta charset="utf-8">
       <link rel="shortcut icon" href="https://www.getpostman.com/favicon.ico">
       <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
-      <link rel="stylesheet" href="../../css/docson.css">
-      <link rel="stylesheet" type="text/css" href="../../css/styles.css">
+      <link rel="stylesheet" href="{{{root}}}/css/docson.css">
+      <link rel="stylesheet" type="text/css" href="{{{root}}}/css/styles.css">
```
Comment (codenirvana) [inline dist/index.mustache:9]: Same change required in `table_of_contents.mustache` CSS path as well.

**Review by codenirvana — CHANGES_REQUESTED** [review]
Directory structure to also follow `/:entity/:format/:version/:draft/` format?

File: dist/index.mustache (hunk 1 above)
Comment (kunagpal) [reply dist/index.mustache]: @codenirvana Yes, to be taken up in a separate pull request, which will be non-blocking on consumers.

**Comment by shamasis** [comment]
How about completely decoupling the web directory structure and the schema directory structure?

**Comment by kunagpal** [comment]
@shamasis The schema and generated web directory structures are essentially decoupled at this point.

**Review by codenirvana — APPROVED** [review]

**Review by shamasis — APPROVED** [review]

**Pull request merged**